rebuild/config/_typography.scss

```diff
@@ -7,7 +7,7 @@ $font--secondary: 'Le Monde Courrier Std', 'Century Schoolbook', serif;
 // Base font weights
 $light: 300;
 $regular: 400;
-$semi-bold: 500;
+$semi-bold: 600;
```

Comment:
Will need to update the Open Sans import so we've got 600 available pls

Reply:
Good spot, this has been updated too.